rest/server_context_test.go

```diff
@@ -238,6 +238,15 @@ func TestGetOrAddDatabaseFromConfig(t *testing.T) {
 	assert.NoError(t, err, "No error while trying to get the existing database name")
 	assert.Equal(t, rosmar.InMemoryURL, dbContext.BucketSpec.Server)
 	assert.Equal(t, bucketName, dbContext.BucketSpec.BucketName)
+
+	dbConfig = DbConfig{
```

Comment:
add comment describing what this part of the test is doing - it's not immediately obvious outside of the context of this PR

```suggestion
	// config with disallowed allow_conflicts=true
	dbConfig = DbConfig{
```